src/utils/event.ts

```diff
@@ -48,16 +48,29 @@ export function decodeEventLog(eventsLog: EventItem[]): EventItem[] {
 export type argType = string | argType[];
 
 // NOTE: argTypes must not contain `uint` , it should be `uint256` instead
-export function warpEventCanonicalSignaturehash(eventName: string, argTypes: argType[]): string {
+export function warpEventCanonicalSignaturehash256(
```

Comment:
We need the event decode and event encode functions as well

Reply:
will add in a separate PR

Reply:
will add in a separate PR